packages/xml-views-validation/src/validators/document/non-unique-id.ts

```diff
@@ -91,15 +97,13 @@ class IdsCollectorVisitor implements XMLAstVisitor {
 // We only care about UI5 elements/controls IDs when check non-unique IDs
 // `id` attributes in these: **known** namespaces which are sometimes used
 // in UI5 xml-views are definitively not relevant for this validation
-const whiteListedNamespaces: Record<string, boolean> = {
-  "http://www.w3.org/1999/xhtml": true,
-  "http://www.w3.org/2000/svg": true,
-  "http://schemas.sap.com/sapui5/extension/sap.ui.core.template/1": true,
-};
+const whiteListedNamespaces: Record<string, boolean> = {};
+whiteListedNamespaces[SVG_NS] = true;
+whiteListedNamespaces[TEMPLATING_NS] = true;
+whiteListedNamespaces[XHTML_NS] = true;
 
 function isNoneUI5id(attrib: XMLAttribute): boolean {
   const parentElement = attrib.parent;
-  const parentPrefix = parentElement.ns ?? DEFAULT_NS;
-  const parentResolvedNamespace = parentElement.namespaces[parentPrefix];
-  return whiteListedNamespaces[parentResolvedNamespace] === true;
+  const parentResolvedNamespace = resolveXMLNS(parentElement) ?? "";
+  return whiteListedNamespaces[parentResolvedNamespace];
```

Comment:
This will return `undefined` instead of `false` if the key doesn't exist

Reply:
thanks

Reply:
This would still return `undefined` instead of `false` for non-whitelisted namespaces